met-web/src/components/common/ControlledInputComponents/ControlledTextField.tsx

```diff
@@ -6,17 +6,17 @@ type IFormInputProps = {
     name: string;
 } & TextFieldProps;
 
-const ControlledTextField: FC<IFormInputProps> = ({ name, ...otherProps }) => {
+const ControlledTextField: FC<IFormInputProps> = ({ name, defaultValue, ...otherProps }) => {
     const {
         control,
-        formState: { errors, defaultValues },
```

Comment:
This works, but react hook form has already a way to manage default values via the defaultValues prop here. The way to use is to revert the changes in this file and to pass {whatever, defaultValues: {...}} to the useForm hook in the AddContactDrawer

Reply:
I did the changes requested but it doesnt not seem to be updating correctly